Sources/UInt128/UInt128.swift

```diff
@@ -42,7 +42,10 @@ public struct UInt128 {
     /// Internal value is presented as a tuple of 2 64-bit
     /// unsigned integers.
     internal var value: (upperBits: UInt64, lowerBits: UInt64)
-
+    
+    public var upperBits: UInt64 { value.upperBits }
+    public var lowerBits: UInt64 { value.lowerBits }
```

Comment:
I purposefully kept these hidden as they are internal details of how this library works and I didn't want to risk having someone depend upon their presence in case the implementation changes in the future. Is there a particular reason why you think these should be exposed now?

Reply:
This is an unrelated change. I'll revert it later.

Reply:
I'll resolve this thread once that revert is introduced.